src/coreclr/jit/emit.cpp

```diff
@@ -5391,7 +5953,7 @@ unsigned emitter::emitEndCodeGen(Compiler* comp,
     // Fill in eventual unused space, but do not report this space as used.
     // If you add this padding during the emitIGlist loop, then it will
     // emit offsets after the loop with wrong value (for example for GC ref variables).
-    unsigned unusedSize = emitTotalCodeSize - emitCurCodeOffs(cp);
+    unsigned unusedSize = emitTotalCodeSize - actualCodeSize;
```

Comment:
Can you add a JITDUMP for this? I'd like to make it more prominent.

(also eventually a metric that jit-analyze knows about)

Reply:
Yes, once this PR goes in, i have a follow-up PR in jitutils to add the `allocSize` and `codeSizeToAllocSizeRatio` metric.

Reply:
Also, the JITDump is already there on line 5953, I will probably use `actualCodeSize` in that instead.